easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+
+name = 'icc'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_2016_composer_edition_for_cpp_2016.0.109.tgz']
```

Comment:
Yes, I renamed it,while it doesn't have a proper version number.
I have the same md5:
f57a892fb494db3c80f20a88aa3e901f  parallel_studio_xe_2016_composer_edition_for_cpp_2016.0.109.tgz